opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbDao.java

```diff
@@ -141,12 +141,18 @@
 
     @org.springframework.beans.factory.annotation.Value("${tip.wlan.ovsdb.listenPort:6640}")
     private int ovsdbListenPort;
+    
+    @org.springframework.beans.factory.annotation.Value("${tip.wlan.ovsdb.externalPort:6640}")
+    private int ovsdbExternalPort;
 
     @org.springframework.beans.factory.annotation.Value("${tip.wlan.mqttBroker.address.external:testportal.123wlan.com}")
     private String mqttBrokerAddress;
 
     @org.springframework.beans.factory.annotation.Value("${tip.wlan.mqttBroker.listenPort:1883}")
     private int mqttBrokerListenPort;
```

Comment:
We should remove this variable, if it's not being used in the class and to avoid ambiguity.

Reply:
Thank you @rahulsharma2020

Reply:
> I think we need maybe some further discussion here, suggest we have a quick talk with yourself and Yong maybe, to make sure we are doing this correctly

Sure @mikehansen1970 , I will ping Yong to see if we could have a discussion